tg/controllers/__init__.py

```diff
@@ -60,6 +62,10 @@ def handle_msgs(self):
             elif keys == 'G':
                 if self.model.jump_bottom():
                     self.refresh_msgs()
+            elif keys == 'd':
```

Comment:
Can we use dd, like in vim.
This way we will reduce accidental msg deletion and make it more user friendly (vim friendly)?

Reply:
done